src/modules/client/components/clientAlerts/CreateClientAlertDialog.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Alert,
+  Box,
+  DialogContent,
+  DialogProps,
+  DialogTitle,
+} from '@mui/material';
+import CommonDialog from '@/components/elements/CommonDialog';
+import StaticForm from '@/modules/form/components/StaticForm';
+import { clientBriefName } from '@/modules/hmis/hmisUtil';
+import {
+  ClientAlertInput,
+  ClientWithAlertFieldsFragment,
+  CreateClientAlertDocument,
+  CreateClientAlertMutation,
+  CreateClientAlertMutationVariables,
+  StaticFormRole,
+} from '@/types/gqlTypes';
+
+interface ClientAlertDialogProps extends DialogProps {
+  client: ClientWithAlertFieldsFragment;
+}
+export const CreateClientAlertDialog: React.FC<ClientAlertDialogProps> = ({
+  client,
+  ...props
+}) => {
+  const clientName = clientBriefName(client);
```

Comment:
Just checking it's appropriate to use `clientBriefName` rather than `clientNameAllParts` here?

Reply:
Yep!